### ISSUE TITLE
pfcmd: Have a way to quickly identify services status

### ISSUE THREAD

**Issue by dwlfrth** [inverse-inc/packetfence#1553]
A simple way to get a return code which will let know if everything that need to be started is started.

bin/pfcmd service pf status is just too... hungry ?

**Comment by louismunro** [comment]
I think we need to reconsider the whole way we deal with processes.
To begin with, it does not make sense anymore to start processes sequentially.
That takes way too long.

But I digress. Yes, we need a better way of polling status for services.

**Comment by julsemaan** [comment]
Lets see wat happens in #1556 and readdress this

**Comment by jrouzierinverse** [comment]
Further thinking about this.
Delaying the autoloading will not fix the issue.
It would require reworking the service managers action to be lighter.

**Comment by nqb** [comment]
> A simple way to get a return code which will let know if everything that need to be started is started.
> 
> bin/pfcmd service pf status is just too... hungry ?

Completely agree with that point of view.

With this feature, we can check by a simple command a health state of a pf server. Return code could be use in scripts to automate upgrade or maintenance for example.

**Comment by nqb** [comment]
I think we should rely on monit to accomplish our needs. Monit is made for that, not `pfcmd`. Moreover, it will more simple to write a monit script rather than change `pfcmd` code,

**Comment by julsemaan** [comment]
This has already been addressed through our systemd integration I believe and completed by @fdurand by reworking the pfcmd service pf status

So I think the original issue is actually fixed

**Comment by nqb** [comment]
> So I think the original issue is actually fixed

@julsemaan: not agree. Currently, how do you check against a return code if "everything that need to be started is started."  ?

`pfcmd service pf status` give us a visual display, not a return code.

**Comment by julsemaan** [comment]
@nqb:

Right, it doesn't return a status code. We could adjust it for that.

There is also the possibility of looking at the status of packetfence.target or packetfence-cluster.target
